assets/js/base/components/sort-select/style.scss

```diff
@@ -1,5 +1,6 @@
 .wc-block-sort-select {
 	margin-bottom: $gap-small;
+	text-align: left;
```

Comment:
Could we move this declaration to `.wc-block-product-sort-select` in `assets/js/base/components/product-sort-select/style.scss`? It doesn't make any visual difference but I think it's better to move it to the most specific selector that needs it (since `.wc-block-sort-select` also affects the reviews' sort select).